Python/susankoju/task3/scraper.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python3
+import os
+import csv
+import json
+import yaml
+import sqlite3
+import argparse
+import requests
+from bs4 import BeautifulSoup
+from multiprocessing import Pool
+import xml.etree.ElementTree as ET
+
+import sys
+sys.setrecursionlimit(25000)
+
+from db import saveProduct
+
+DEFAULT_MAX = 50
+OUTPUT_FOLDER = 'output'
+DEFAULT_SEARCH = 'computer'
+DEFAULT_FILENAME = 'results'
+BASE_URL = 'https://hamrobazaar.com/'
+BASE_SEARCH_URL = BASE_URL + 'search.php'
+
+def mapTableData(table):
```

Comment:
```suggestion
def map_table_data(table):
```
Please use sake_case naming convention